Seaography/docs/01-index.md

```diff
@@ -0,0 +1,82 @@
+# Index
+
+## Project Description
+
+Seaography is a CLI tool that generates a ready to compile Rust GraphQL API for a relational database.
+
+The tool connects to a relational database, explores its schema and based on it generates the ORM entities and GraphQL queries.
+
+## Benefits
+
+* Generate the boilerplate
+* Focus on business logic
+* Easy and fast to get started
+
+
+## Current features
+
+* Query entities with filters and relations
+* Dataloader optimized to solve 1+N problem
+
+## Upcoming features
+* Procedural macro API to reduce code surface
+* Single entity queries with relations
+* Create mutations
+* Update mutations
+* Delete mutations
+
+## Generated code dependencies
+
+In order to understand how the generated code works its recommended to study the following resources:
+
+1. [async-graphql](https://docs.rs/async-graphql/latest/async_graphql/)
+
+  Is a server side GraphQL library for Rust.
+
+2. [sea-orm](https://docs.rs/sea-orm/latest/sea_orm/)
+
+  SeaORM is a relational ORM to help you build web services in Rust with the familiarity of dynamic languages.
+
+3. [poem](https://docs.rs/crate/poem/latest)
+
+  A full-featured and easy-to-use web framework with the Rust programming language.
+
+4. [tokio*](https://docs.rs/tokio/latest/tokio/)
+
+  A runtime for writing reliable, asynchronous, and slim applications with the Rust programming language.
+
+5. [itertools*](https://docs.rs/itertools/latest/itertools/)
+
+  Extra iterator adaptors, functions and macros.
+
+* Recommended to study, but not required
+
+## How to use
```

Comment:
I am hoping to see a more comprehensive step by step guide, from the sample database to running GraphQL queries, perhaps with a few screenshots along the way if that help illustrate. Actually, if you have the energy, a short video screencast would be perfect! (It's not necessary though)

But yeah at least you get the idea

Reply:
Great point.

I will create a page "How to run the example database" and document the step by step process using pictures. 

Regarding the video I will take a note of it and work on it on the future.